json-rpc/types/src/views.rs

```diff
@@ -1091,6 +1091,8 @@ pub enum TransactionDataView {
         script_bytes: BytesView,
         script: ScriptView,
     },
+    #[serde(rename = "statecheckpoint")]
+    StateCheckpoint {},
```

Comment:
why it needs the {} part?

Reply:
I was just matching the `WriteSet {}` variant above..